pkgs/tools/misc/qmk/default.nix

```diff
@@ -3,38 +3,59 @@
 , fetchpatch
 }:
 
-python3.pkgs.buildPythonApplication rec {
+let
+  inherit (python3.pkgs) buildPythonApplication fetchPypi;
+in
+buildPythonApplication rec {
   pname = "qmk";
-  version = "0.0.45";
+  version = "0.0.52";
 
-  src = python3.pkgs.fetchPypi {
+  src = fetchPypi {
     inherit pname version;
-    sha256 = "43f297f36b21d68c34c5efa0ce1449dddb2e28753f80939cadf761ee7a2a0901";
+    sha256 = "sha256-mNF+bRhaL6JhNbROmjYDHkKKokRIALd5FZbRt9Kg5XQ=";
   };
 
-  patches = [
-    # https://github.com/qmk/qmk_cli/pull/48
-    (fetchpatch {
-      name = "remove-unused-install-requires.patch";
-      url = "https://github.com/qmk/qmk_cli/commit/75b6ada1feccfa5a9bc2bb07a4cc749ef40d02dd.patch";
-      sha256 = "0lwi1dz35p07vha5gwq2jxm5q49vm99ix4jyhd6g6ypqbq1qiwc8";
-    })
-  ];
-
   nativeBuildInputs = with python3.pkgs; [
+    flake8
+    nose2
+    pep8-naming
     setuptools-scm
+    yapf
```

Comment:
Are those checkInputs? We do not need to run linting tests.

Reply:
Nope. They are requirements.

Reply:
We should instead rebase and carry forward the patch from https://github.com/qmk/qmk_cli/pull/48.  Not only are they not real runtime requirements, they're also pinned to exact versions, which is going to be a maintenance headache.
https://github.com/qmk/qmk_cli/blob/master/setup.py#L67

Reply:
> We should instead rebase and carry forward the patch

Can someone do it?

And why this was not patched yet?

Reply:
We could just remove then with substituteInPlace.

Reply:
A local patch? I am not so sure we should do it if the upstream didn't.